css-canvas-medical/styles.css

```diff
@@ -0,0 +1,276 @@
+}
+
+.column-one-fourth {
+  flex-basis: 25%;
+}
+
+.column-one-third {
+  flex-basis: calc(100% / 3);
+}
+
+.column-one-half {
+  flex-basis: calc(100% / 2);
+}
+
+.column-one-fifth {
+  flex-basis: calc(100% / 5);
+}
+
+.column-fill {
+  flex-grow: 1;
+  flex-shrink: 1;
+  flex-basis: auto;
+}
+
+.nav-bar {
+  justify-content: space-between;
+  padding-bottom: 10px;
+}
+
+.nav-links {
+  gap: 50px;
+  display: flex;
+  justify-content: flex-end;
+  align-items: center;
+}
+
+.nav-links > a {
+  white-space: nowrap;
+}
+
+#logo-blue {
+  width: 120px;
+  height: 30px;
+  margin-left: 30px;
+}
+
+.nav-button {
+  background-color: white;
+  border-style: none;
+  color: #bdbdbd;
+  text-decoration-line: none;
+}
+
+.nav-button:hover {
+  color: black;
+}
+
+#contact-button {
+  display: block;
+  color: white;
+  background-color: #284465;
+  width: 100px;
+  text-align: center;
+  align-self: center;
+  padding: 10px;
+  border-radius: 0.2rem;
+}
+
+#contact-button:hover {
+  background-color: #3d6797;
+}
+
+.blue-button {
+  display: block;
+  color: white;
+  background-color: #284465;
+  width: 200px;
+  text-align: center;
+  align-self: flex-start;
+  padding: 20px;
+  border-radius: 0.2rem;
+  z-index: 3;
+  text-decoration-line: none;
+  font-family: Avenir, sans-serif;
+  font-stretch: expanded;
+  font-size: 20px;
+}
+
+.border {
+  display: block;
+  border-bottom: 3px solid rgb(229, 231, 227);
+  margin-bottom: 30px;
+}
+
+.hero-container {
+  max-width: 1300px;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  margin: 0 auto;
+  height: 100vh;
+  position: relative;
+}
+
+.hero-image {
+  max-width: 1300px;
+  z-index: -1;
+}
+
+.hero-text-container {
+  margin-top: 0;
+  position: absolute;
+  width: 60%;
+  left: 3%;
+}
+
+.hero-text {
+  font-family: 'Tiempos Headline', sans-serif;
+  text-align: left;
+  color: white;
+  z-index: 100;
+}
+
+h1.hero-text {
+  color: white;
+  font-size: 50px;
+  margin-bottom: 3rem;
+}
+
+p.hero-text {
+  margin-top: 0;
+  font-size: 15px;
+  font-family: Arial, Helvetica, sans-serif;
+  font-stretch: ultra-expanded;
+  line-height: 2;
+  letter-spacing: 2px;
+  margin-bottom: 20px;
+}
+
+.hero-overlay {
+  background: linear-gradient(transparent, rgba(0, 0, 0, 0.5));
+  position: absolute;
+  top: 0;
+  bottom: 0;
+  left: 0;
+  right: 0;
+  z-index: ;
+}
+
+.locations-section {
+  justify-content: column;
+  align-items: center;
+  width: 100%;
+  margin-right: 20px;
+  margin-left: 20px;
+  background-color: #fafafa;
+}
+
+.locations-heading {
+  text-align: center;
+  margin-left: auto;
+  margin-right: auto;
+}
+
+.locations-heading > h2 {
+  color: #7f7f7f;
+}
+
+.locations-heading > p {
+  color: #8f8e8e;
+}
+
+.locations-text-content {
+  display: flex;
+  justify-content: center;
+  text-align: center;
+  margin-bottom: 20px;
+}
+
+.locations-text-content * {
+  list-style: none;
+  line-height: 2rem;
+  background-color: #fafafa;
+  color: #8f8e8e;
+}
+
```

Comment:
Ideally you don't really want to use the `*` selector because it will target _any_ descendent of the element with the class name specified. Instead, maybe consider adding this class to your `ul`s and then target the child `li` to be more direct and specific.